src/tests/test_csv_parser.cpp

```diff
@@ -0,0 +1,95 @@
+#include <cstddef>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "all_tables_config.h"
+#include "parser/csv_parser/csv_parser.h"
+
+namespace tests {
+
+using ::testing::ContainerEq;
+
+class TestCSVParser : public ::testing::Test {};
+
+static void CheckGetNextRow(tests::TableConfig const& table, std::size_t num_rows,
```

Comment:
Why do we need `num_rows` here? It seems like `expected.size()` already has this information?